GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -214,34 +216,59 @@ public void CanFetchAndCheckoutAfterDeletingIndex()
                 .WithDeepStructure(FileSystemRunner.DefaultRunner, this.fastFetchControlRoot);
         }
 
+        // Each file listed in in ls-files --debug outputs a structure that looks like so 
+        //  nuget.config
+        //  ctime: 1540245129:777055100
+        //  mtime: 1541723805:620569300
+        //  dev: 0 ino: 0
+        //  uid: 0 gid: 0
+        //  size: 617 flags: 0
+        //
+        // The index that FastFetch crafts fills the dev/ino/uid/gid fields of the index with 0 for speed
+        // Git for Windows writes these fields out as 0, but once a Unix version of Git writes out the index,
+        // they get filled with actual values. This breaks our Index comparison logic, so we'll remove these 

```

Comment:
> but once a Unix version of Git writes out the index, they get filled with actual values

Does FastFetch need to use actual values (on Mac)?  What is the impact if it does not?